### ISSUE TITLE
Pull into Craft from Readwise: highlights sharing the same tag EOM

### ISSUE THREAD

**Issue by saumyamoon** [TfTHacker/craft42-readwise#1]

**Comment by TfTHacker** [comment]
Good idea. How would this work? Would you want to see a list of tags, select the list and then all highlights with those tags return? Would it save that list to re-run the extract?

Also what details for each highlight would be imported?

**Comment by saumyamoon** [comment]
Thanks. 
- See a list of all available tags and be able to choose a single or multiple tags for import  
- It would be great to be able to save multiple-tags as a group for re-import, but not a critical function
- Import all details (book title, author, location in Kindle or page). Absence of these would be a deal-breaker for me. 
- I don't care so much about cover images, although others might.  

Thanks!!!
